### ISSUE TITLE
`guardian` in `ops_multisig_old` is redundant

### ISSUE THREAD

**Issue by petrovska-petro** [Badger-Finance/badger-multisig#1294]
Currently the `guardian` in [governance-timelock](https://etherscan.io/address/0x21CF9b77F88Adf8F8C98d7E33Fe601DC57bC0893#readContract#F6) is a deprecated msig, where the only owner is `devmsig` as of now.

This configuration kind of defeats the purpose of guardian which has rights to *cancel* transaction that has being queued, while the admin and the guardian are exactly the same addr.

Considerations:

- [ ] update the guardian

**Comment by petrovska-petro** [comment]
probably worthy to be batch whenever #1277 is posted

**Comment by sajanrajdev** [comment]
I agree on changing the guardian to a live Multisig and it should be one with very little overlap with Dev, for instance TechOps. The reason for this is that the guardian should be able interfere vetoing malicious transactions in case that dev is compromised.

I would defer to @GalloDaSballo and @dapp-whisperer for the best configuration though.

**Comment by GalloDaSballo** [comment]
@dapp-whisperer if we want to be super cautious guardian could be a List of EOAs 
Else can use techops or similar

No strong opinion until I understand the goal of the timelock

**Comment by petrovska-petro** [comment]
the goal of the TL is to put the minting/burning actions of badger token behind that
currently it could happen atomically by the devmsig
and to keep higher sec standard we want to keep it behind TL at least
cc: @GalloDaSballo

**Comment by gosuto-inzasheru** [comment]
edit: `setGuardian` does not need to occur through a timelock, so the guardian _cannot_ protect against a malicious dev msig. only make cancelling easier.
